iceoryx2/src/config.rs

```diff
@@ -394,10 +392,18 @@ impl Config {
     /// [`Config::setup_global_config_from_file()`]
     /// is called after this function was called, no file will be loaded since the global default
     /// config was already populated.
+
     pub fn global_config() -> &'static Config {
         if !ICEORYX2_CONFIG.is_initialized()
             && Config::setup_global_config_from_file(unsafe {
-                &FilePath::new_unchecked(DEFAULT_CONFIG_FILE)
+                &FilePath::new_unchecked(
+                    dirs::config_dir()
+                        .unwrap()
```

Comment:
Won't this panic if the directory doesn't exist. The `None` case should be handled explicitly.